projects/storefrontlib/src/cms-structure/routing/default-routing-config.ts

```diff
@@ -89,6 +89,9 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
   budgetEdit: {
     paths: ['organization/budget/edit/:code'],
   },
+  orderApprovals: {
```

Comment:
Perhaps we could name it `orderApprovalList` for improved readability.  Also, let's declare it next to the orderApprovalDetails route.